doc/release-notes/rl-2511.section.md

```diff
@@ -54,6 +54,10 @@
 - `gramps` has been updated to 6.0.0
   Upstream recommends [backing up your Family Trees](https://gramps-project.org/wiki/index.php/Gramps_6.0_Wiki_Manual_-_Manage_Family_Trees#Backing_up_a_Family_Tree) before upgrading.
 
+- The `clickhouse` package now track the stable upstream version per [upstream's
+  recommendation](https://clickhouse.com/docs/faq/operations/production). Users
+  can continue to use the `clickhouse-lts` package if desired.
```

Comment:
```suggestion
- The `clickhouse` package now track the stable upstream version per [upstream's recommendation](https://clickhouse.com/docs/faq/operations/production).
  Users can continue to use the `clickhouse-lts` package if desired.
```
Lets write one sentence per line, so that future editors have it easy.

Reply:
This is actually how Helix auto-wraps it with its `:reflow` command.